samples/cellular/nrf_cloud_multi_service/prj.conf

```diff
@@ -7,7 +7,7 @@
 # Log level
 # For more verbose and detailed log output, set the log level to
 # CONFIG_MULTI_SERVICE_LOG_LEVEL_DBG=y instead.
-CONFIG_MULTI_SERVICE_LOG_LEVEL_INF=y
+CONFIG_MULTI_SERVICE_LOG_LEVEL_DBG=y
```

Comment:
These `prj.conf` changes are for testing and development. I will remove them before leaving draft status.